src/Formatter/CallableFormatter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace League\BooBoo\Formatter;
+
+use Exception;
```

Comment:
Good point that I didn't import it in the other. That's because I used PHPStorm "add method stubs" functionality to quickly generate the method stub. It just copied the type hints from the `HandlerInterface`.
![screen shot 2015-03-03 at 22 19 26](https://cloud.githubusercontent.com/assets/1410358/6472521/6db85e6c-c1f3-11e4-992d-84b0646c3c4f.png)